test/unit-test/CMakeLists.txt

```diff
@@ -20,6 +37,8 @@ list(APPEND real_include_directories
 # list the directories your test needs to include
 list(APPEND test_include_directories
             ${JSON_INCLUDE_PUBLIC_DIRS}
+            ${UNIT_TEST_DIR}

```

Comment:
I don't think this directory is ever defined anywhere.

Reply:
I see it in the arguments below.  The compile would fail otherwise.
```
[ 91%] Building C object unit-test/CMakeFiles/core_json_utest.dir/core_json_utest_runner.c.o
cd /home/admin/work/coreJSON-dan/build/unit-test && /usr/bin/cc  
-I/home/admin/work/coreJSON-dan/source/include 
-I/home/admin/work/coreJSON-dan/test/unit-test 
-I/home/admin/work/coreJSON-dan/build 
-I/home/admin/work/coreJSON-dan/test/unit-test/Unity/src 
-I/home/admin/work/coreJSON-dan/test/unit-test/Unity/extras/fixture/src 
-I/home/admin/work/coreJSON-dan/test/unit-test/Unity/extras/memory/src 
--coverage -Wall -Wextra -Werror -g -std=gnu90 -o CMakeFiles/core_json_utest.dir/core_json_utest_runner.c.o 
-c /home/admin/work/coreJSON-dan/build/unit-test/core_json_utest_runner.c
```